docs/twig-components/method/excerpt.md

```diff
@@ -1,6 +1,42 @@
 # excerpt
 
-`excerpt(length = 280, includeTitle = false, focus = null, wrap = false)` is a Twig filter to ...
+`excerpt(length = 280, includeTitle = false, focus = null, wrap = false)` is a Twig filter create a short, text-only, 
+excerpt of a record or a string. It's useful to get short blurbs of text
+for overview pages, listings, et cetera. If you pass it a string, it will simply strip out HTML and, reduce it to a 
```

Comment:
There is a typo on et cetera. It should be etcetera.

Reply:
You can write it both ways... 
As well et cetera as etcetera are accepted 👍

Reply:
All right!! You taught me something new. I always thought the "all words together" was the only and good way. Thanks 👍